### DISCUSSION TITLE
Support for SAMD21 mcu

### DISCUSSION THREAD

**Discussion by sslupsky** [feilipu/Arduino_FreeRTOS_Library#47] [Q&A]
Any plans to support the samd21 mcu?

**Comment by feilipu** [comment]
The [SAMD21](https://github.com/BriscoeTech/Arduino-FreeRTOS-SAMD21) is already supported by FreeRTOS. 👍 

But since the SAMD21 is not an AVR device, it is unlikely that this repo will support it.

**Comment by sslupsky** [comment]
Unfortunately, FreeRTOS does not fully support Tickless for SAMD21 or any other core M0 device for that matter. Thanks for the feedback.
